internal/corazawaf/transaction.go

```diff
@@ -1372,6 +1373,11 @@ func (tx *Transaction) AuditLog() *auditlog.Log {
 
 	clientPort, _ := strconv.Atoi(tx.variables.remotePort.Get())
 	hostPort, _ := strconv.Atoi(tx.variables.serverPort.Get())
+	args := &strings.Builder{}
+	for _, arg := range tx.Collection(variables.Args).FindAll() {
```

Comment:
I truly doubt the internal transaction has to know about all these fields. I wonder if this could happen in the internal/auditlog#Transaction type.

Reply:
I was able to move this into formats_ocsf.go.